unity-renderer/Assets/ABConverter/Core.cs

```diff
@@ -802,12 +802,12 @@ private void MarkShaderAssetBundle()
             ABConverter.Utils.MarkAssetForAssetBundleBuild(env.assetDatabase, mainShader, MAIN_SHADER_AB_NAME);
         }
 
-        internal virtual void InitializeDirectoryPaths(bool deleteIfExists)
+        internal virtual void InitializeDirectoryPaths(bool deleteDownloadDirIfExists, bool deleteABsDireIfExists)
```

Comment:
```suggestion
        internal virtual void InitializeDirectoryPaths(bool deleteDownloadDirIfExists, bool deleteABsDireIfExists)
```
This signature change is completely justified? I'm looking at the code and observing that both arguments are called always with the same value.

Reply:
Check [this line](https://github.com/decentraland/unity-renderer/blob/a9ed5d9f6dde043ae85c5ef31db412ef359db1f8/unity-renderer/Assets/ABConverter/Client.cs#L532), we need to avoid deleting the ABs folder on every `DumpWearablesCollection()` call to keep the ABs from the previously-converted collections. Otherwise, after dumping all the collections sequentially, we end up with only the ABs from the latest collection in the output directory.